src/Grid/grid.jl

```diff
@@ -132,12 +132,13 @@ reference_facets(::Type{<:AbstractRefShape})
 @inline reference_facets(refshape::Type{<:AbstractRefShape{3}}) = reference_faces(refshape)
 
 """
-    geometric_interpolation(::AbstractCell)::Interpolation
+    geometric_interpolation(::AbstractCell)
+    geometric_interpolation(::Type{<:AbstractCell})
 
 Each `AbstractCell` type has a unique geometric interpolation describing its geometry.
-This function returns that interpolation.
+This function returns that interpolation, which is always a scalar interpolation.
```

Comment:
Do we really want to enforce this behavior?

Also, since I just realize, we probably should double check the internal implementation where this is used to see if it might break when interacting with embedded elements.

Reply:
Since we are giving only the cell, we don't know anything about the spatial dimension, and therefore we cannot return correct vectorized interpolation in order to support embedded elements. 

`default_geometric_interpolation(ip::Interpolation)` returns a `Vectorized` interpolation with `fielddim = refdim`, which is used to give a default value for `AbstractValues`